### ISSUE TITLE
Video recording issue - scrambled images

### ISSUE THREAD

**Issue by gu-ma** [timscaffidi/ofxVideoRecorder#56]
Not sure if this is a issue related to ofxVideoRecorder or to ffmpeg itself.

I'm using ofxVideoRecorder in a project and I'm having some strange issues. When I start running the application everything is fine and all videos are recorded perfectly but after some time some of the vids are completely scrambled, like that:

![image](https://user-images.githubusercontent.com/4388440/29294020-a03fb2e2-814d-11e7-8c67-6d9b641d8db8.png)

It does not always happen and sometime it happens after 30 mins sometime after an hour or more.

The program does not use much memory but it's quite CPU intensive. I've tried to make some of the processes threaded and the CPU usage went down but the bug did not disappear... I've also tried most of the codec available: mjpeg, h264, mpeg4 and even hap but at some point this bug keeps reappearing.

Did anyone run into a similar problem? Any suggestions?

**Comment by stephanschulz** [comment]
I see the same problem.
have you ever figured out why this happens?

**Comment by gu-ma** [comment]
Hum, that was a long time ago, I don't really remember... 
I think the problem came from the video codec I tried different ones at the end. Some were working better than others.

**Comment by stephanschulz** [comment]
thanks for the reply.
for me to it works 90% of the time.
after rebooting the computer it's ok again.
 
I am using:
VideoCodec: libx264
VideoBitrate: 2000k
PixelFormat: rgb24
OutputPixelFormat: yuv420p

**Comment by gu-ma** [comment]
Yes, It was also working fine at the beginning and getting bad after some time. 

I was also fiddling with some of those settings... Can't recall which ones ended up working the best though 🤷‍♂️
